index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main id="main">
+     <h1 id="title">MUHAMMED ALI</h1> 
+      <p>The American professional boxer</p>
+      <section id="tribute-info">
+        <h3 id="headline">Here's a time line of Muhammed life:</h3>
+        <ul>
+          <li><strong>1914</strong> - he was known for his social message of black pride and black resistance to white domination and for refusing induction into the U.S. Army during the Vietnam War.24 Aug 2022</li>
+          <li>
+            <strong>1933</strong> - Leaves his family's farm to attend the
+            University of Minnesota, thanks to a Depression era program known as
+            the "National Youth Administration"
+
```

Comment:
- [x] Kindly add the missing li closing tag in this block.

Reply:
@Ofarouq310, Kindly state as to why it is necessary to have a closing tag for all your HTML elements ✌️